common/repo/GitRepo.java

```diff
@@ -21,17 +21,24 @@
 import com.google.auto.factory.AutoFactory;
 import com.google.auto.factory.Provided;
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
 import com.google.startupos.common.repo.Protos.Commit;
 import com.google.startupos.common.repo.Protos.File;
 import java.io.IOException;
 import java.nio.file.FileSystem;
 import java.nio.file.Paths;
+import java.util.Date;
+import java.util.List;
+
+import jdk.nashorn.internal.ir.annotations.Immutable;
```

Comment:
Oops, I think the wrong `@Immutable` annotation was imported here. Did you by any chance mean to import error-prone's annotation rather than the JDK's internal one for Nashorn? :)